src/main/java/nl/mrwouter/minetopiafarms/commands/MTFarmsCMD.java

```diff
@@ -56,16 +59,11 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
 
 			NPC npc = CitizensAPI.getNPCRegistry().createNPC(EntityType.PLAYER, Main.getMessage("NPC.Name"));
 
-			npc.data().set(NPC.PLAYER_SKIN_UUID_METADATA, Main.getMessage("NPC.Skin.UUID"));
+			npc.data().setPersistent("player-skin-name", Main.getMessage("NPC.Skin.Name"));
```

Comment:
Uit interesse, waarom is dit zo gedaan? Ik heb de despawn zo ingebouwd omdat dit vroeger zo gedocumenteerd was volgens mij. Is dit veranderd?